perf/impl/go-libp2p/v0.28/Makefile

```diff
@@ -3,11 +3,10 @@ GO_FILES := $(wildcard *.go)
 all: perf
 
 perf: $(GO_FILES)
-	docker run --rm --user "$(shell id -u):$(shell id -g)" -v "$(shell pwd)":/usr/src/myapp -w /usr/src/myapp -e GOCACHE=/usr/src/myapp/.cache golang:1.20 go build -o perf .
+	docker run --rm --user "$(shell id -u):$(shell id -g)" -v "$(shell pwd)":/usr/src/myapp -w /usr/src/myapp -e GOCACHE=/usr/src/myapp/.cache golang:$(shell awk '/^go [0-9]+(\.[0-9]+)?$$/ {print $$2}' go.mod) go build -o perf .
```

Comment:
By getting the version from `go.mod` we don't have to update the Makefile each time.